src/toxcore/messenger/packet/user_status.rs

```diff
@@ -0,0 +1,80 @@
+/*! UserStatus struct.
+*/
+
+use nom::le_u8;
+
+use crate::toxcore::binary_io::*;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+/// Status of user
```

Comment:
Not very important, but we put derives after docs everywhere. So I think it will look better here too.

Reply:
done